docker-builds/Dockerfile.db-preloaded

```diff
@@ -5,6 +5,8 @@ FROM metasfresh/metas-db:$REFNAME as db
 # https://cadu.dev/creating-a-docker-image-with-database-preloaded/
 # dirty but works
 
+# TODO: see how RunMigrationScriptsTest.java does it using our own java-tool. It's better for different reasons.
```

Comment:
@metas-ts (loud thinking)

u can even consider having a dedicated repo for `de.metas.migration` build it and deploy it as a docker image.
Then on this docker build we can grab the built jar from there.